source/_integrations/myq.markdown

```diff
@@ -20,6 +20,7 @@ ha_platforms:
 ha_dhcp: true
 ha_integration_type: integration
 ---
+## This integration is broken and unlikely to ever be reliable. Chamberlain / LiftMaster / MyQ is hostile to 3rd party use of their API. Consider using a different brand if you want a garage door opener that works with Home Assistant.
```

Comment:
There are better places for these things. We have alerts available if that is needed and are considering at this point.

Please track the issue tracker for status updates.

Thanks! 👍 

../Frenck

Reply:
The availability of the integration is misleading and will result in people buying a product that actually does not work with the expectation that it does work. A warning should be added for sure.

Reply:
Thanks @lewisdiamond, however, see my previous answer, thanks 👍